ui/desktop/electron-app/src/helpers/app-updater.js

```diff
@@ -19,6 +20,33 @@ if (debug && process.env.APP_UPDATER_CURRENT_VERSION) {
   currentVersion = process.env.APP_UPDATER_CURRENT_VERSION;
 }
 
+const returnArchitectureToUpdate = () => {
+  const nodeArchitecture = process.arch;
+
+  if (nodeArchitecture === 'arm64') {
+    return nodeArchitecture;
+  }
+
+  if (nodeArchitecture === 'amd64') {
```

Comment:
I don't think you need this if check since if it's not `arm64`, we can assume the process is now amd64